### PR TITLE
Controlpanels Endpoint

### PR DESCRIPTION
**Pull request by tisto**

### CODE REVIEW

**Comment by jaroel** [comment]
Refs #342

File: src/plone/restapi/tests/test_documentation.py [hunk 1]
```diff
@@ -837,6 +844,20 @@ def test_history_revert(self):
         response = self.api_session.patch(url, json={'version': 0})
         save_request_and_response_for_docs('history_revert', response)
 
+    @unittest.skipIf(not PLONE5, 'Just Plone 5 currently.')
```
Comment (tisto) [inline src/plone/restapi/tests/test_documentation.py]: @jaroel how can we retrieve the controlpanel request/response if we just run the tests on Plone 4?

File: docs/source/_json/controlpanels_get.resp [hunk 2]
```diff
@@ -0,0 +1,15 @@
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+[
+  {
```
Comment (tisto) [inline docs/source/_json/controlpanels_get.resp]: @jaroel how do you make it possible that you just show the mail and editing control panel and not all available control panels?

Comment (jaroel) [reply docs/source/_json/controlpanels_get.resp]: The available controlpanels in the rest API are define in the service/controlpanels/configure.zcml, as most -but not all- controlpanels are registry based.

Comment (tisto) [reply docs/source/_json/controlpanels_get.resp]: @jaroel that's why I am asking, we have more than two controlpanels listed there and just two show up in the response.

**Comment by tisto** [comment]
LGTM

**Pull request merged**